### PR TITLE
Change structure & add form.html with styles

### PR DESCRIPTION
**Pull request by nikokreissl**
- created sub-dirs for css and html files und updated links accordingly
- created form.html and applied some styles to it
- changed that buttons are globally controlled by tag and not by class

### CODE REVIEW

File: components/html/form.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          <label for="tags">Tag:</label>
+          <input type="text" name="tags" id="tags" />
+        </div>
+          <button type="submit">Submit question</button>
+        </div>
+      </form>
+    </main>
+
+    <nav data-js="navigation">
+      <ul class="nav-items" data-js="navigation-items">
```
Comment (sahedw) [inline components/html/form.html:42]: Maybe according to BEM "nav__list" as a class? "items" could be needed later for the individual item-manipulation within the list.

**Comment by sahedw** [comment]
Really good job! Looks clean and organized. Just added a small comment regarding the class name. 

There is also a problem with your profile picture on the user page. I saw that you reorganized your project folder and maybe there is something that needs to be updates within the html file. 

Other than that: lgtm!

File: components/html/form.html (hunk 1 above)
Comment (nikokreissl) [reply components/html/form.html:42]: Good point! I will change and push into this PR 😊

File: components/css/global.css [hunk 2]
```diff
@@ -27,10 +27,33 @@ a:hover {
...
+
 @media (max-width: 600px) {
   :root {
     font-size: small;
   }
+
+  button {
+    padding: 10px;
+    font-size: 1.1em;
+    margin: 5px 0px 5px 0px;
```
Comment (mpagels) [inline components/css/global.css:54]: Here you could just write `5px 0px` and omit the last two values. It is shorter and have the same effect.

**Review by mpagels — COMMENTED** [review]
Did leave a tiny improvement. Good work here.

**Pull request merged**